schema.json

```diff
@@ -2820,16 +2820,32 @@
     "enable": "smartButtons",
     "settings": [
       {
-        "type": "checkbox",
-        "label": "i18n.ShowPayPalTagline",
-        "force_reload": true,
-        "id": "paymentbuttons-paypal-tagline"
+        "type": "heading",
+        "content": "i18n.PaymentButtonsContainer"
       },
       {
-        "type": "checkbox",
-        "label": "i18n.ShowFundingIcons",
+        "type": "select",
+        "label": "i18n.ButtonContainer",
         "force_reload": true,
-        "id": "paymentbuttons-paypal-fundingicons"
```

Comment:
Don't forget to remove default values for

- paymentbuttons-paypal-fundingicons
- paymentbuttons-paypal-tagline
- paymentbuttons-paypal-size

as they are not used more

Reply:
fixed